scripts/test-ecdhe-rsa-key-exchange-with-bad-messages.py

```diff
@@ -90,21 +107,27 @@ def main():
     else:
         run_only = None
 
+    if not ciphers:
+        ciphers = [CipherSuite.TLS_ECDHE_RSA_WITH_AES_128_CBC_SHA]
```

Comment:
Can't `ciphers` just be initialized with this value instead of `None`?

Reply:
it can, but this is more consistent with the `test-conversation.py` boilerplate